packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.tsx

```diff
@@ -214,24 +218,19 @@ const AccountRecoveryResetPassword = ({
 
       logViewEvent(viewName, 'verification.success');
 
-      switch (integration.type) {
-        // NOTE: SyncBasic check is temporary until we implement codes
-        // See https://docs.google.com/document/d/1K4AD69QgfOCZwFLp7rUcMOkOTslbLCh7jjSdR9zpAkk/edit#heading=h.kkt4eylho93t
-        case IntegrationType.SyncDesktopV3:
-        case IntegrationType.SyncBasic:
-          firefox.fxaLoginSignedInUser({
-            authAt: accountResetData.authAt,
-            email,
-            keyFetchToken: accountResetData.keyFetchToken,
-            sessionToken: accountResetData.sessionToken,
-            uid: accountResetData.uid,
-            unwrapBKey: accountResetData.unwrapBKey,
-            verified: accountResetData.verified,
-          });
-          break;
-        case IntegrationType.OAuth:
-          // TODO: Consider providing a way to redirect user back to RP.
-          break;
+      if (
+        isSyncDesktopV3Integration(integration) ||
+        (isOAuthIntegration(integration) && integration.isSync())
```

Comment:
nit: Could this be pulled into `isSyncOAuthIntegration` helper function?

Reply:
Good call, we reference this a few times now. I guess I'll add it in `models/integrations/oauth-integration.ts`, feels like we should pull some of these into another file.